distributed/__init__.py

```diff
@@ -4,7 +4,7 @@
 from .core import connect, rpc
 from .deploy import LocalCluster, Adaptive, SpecCluster, SSHCluster
 from .diagnostics.progressbar import progress
-from .diagnostics.plugin import WorkerPlugin, SchedulerPlugin, PipInstall
+from .diagnostics.plugin import WorkerPlugin, SchedulerPlugin, PipInstall, UploadFile
```

Comment:
As most users will interact with this plugin via `Client.upload_file`, I don't think we need to include it in the top-level `distributed` namespace

```suggestion
from .diagnostics.plugin import WorkerPlugin, SchedulerPlugin, PipInstall
```

Reply:
:+1: